src/pages/OnRampKitDemo.tsx

```diff
@@ -70,7 +70,7 @@ const OnRampKitDemo = ({ setStep }: OnRampKitDemoProps) => {
 
       <Typography marginTop="16px">
         Allow users to buy cryptocurrencies using a credit card and other payment options directly
-        within your app. Click on "Buy USDC" to on-ramp funds to your Safe using the Stripe widget!
+        within your app. Click on "Buy USDC" to on-ramp funds to your Safe using Monerium or the Stripe widget!
```

Comment:
It's better to remove completely "Click on "Buy USDC" to on-ramp funds to your Safe using Monerium or the Stripe widget!" because in the monerium case you can't after logged in

<img width="546" alt="image" src="https://github.com/5afe/account-abstraction-demo-ui/assets/1576776/c8fbc3a0-7879-4ebe-b293-93bd3342cb70">

Reply:
Good point. I've removed it in 85e4ab148d477a84e0eaf9754a0899c282fc04c5.